manifests/zabbixapi.pp

```diff
@@ -0,0 +1,33 @@
+# @summary This will install the zabbixapi gem.
+#
+# @param zabbix_version This is the zabbix version.
+# @param puppetgem Provider for the zabbixapi gem package.
+class zabbix::zabbixapi (
+  $zabbix_version = $zabbix::params::zabbix_version,
+  $puppetgem = $zabbix::params::puppetgem,
```

Comment:
Should this be `String[1]` for both?

Reply:
In theory, yes.
In reality, we do not enforce these anywhere else, so I find it odd to do it here.

(read: we should do it globally, but not in this PR)